cmd/omni/cmd/cmd.go

```diff
@@ -516,18 +516,18 @@ var initOnce = sync.OnceValue(func() *cobra.Command {
 		"maximal interval between etcd backups",
 	)
 
-	rootCmd.Flags().Int64Var(
+	rootCmd.Flags().Float64Var(
 		&config.Config.EtcdBackup.UploadLimit,
 		"etcd-backup-upload-limit",
 		config.Config.EtcdBackup.UploadLimit,
-		"throughput limit for etcd backup uploads",
+		"throughput limit in MiB for etcd backup uploads, that can be xx.yy",
```

Comment:
MiB feels too non-granular to me, maybe KiB would be the sweet spot?

Reply:
You can specify things like 0.5 or 0.02. But I wonder if it would be better to specify in Mbit (bits) rather than bytes. Usually when we talk about throughput we talk 100 Mbits or 1 Gbit.

Reply:
Another idea - use "byte size" type? Introduce a new cobra flag type, use one of the byte size libraries out there (maybe [this](https://github.com/zpatrick/go-bytesize))? So users can do `10MiB` etc.